### ISSUE TITLE
ME1 PMCheckConditional - ByteProperties missing "GlobalEnum"

### ISSUE THREAD

**Issue by Kinkojiro** [ME3Tweaks/LegendaryExplorer#22]
Getting an error when looking at sequence 4562 RP102_TheFan in BIOA_STA60_02a_DSG.SFM

```
System.ArgumentNullException: Value cannot be null.
Parameter name: key
   at System.Collections.Generic.Dictionary`2.FindEntry(TKey key)
   at ME1Explorer.Unreal.ME1UnrealObjectInfo.getEnumValues(String enumName, Boolean includeNone) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\ME1\Unreal\ME1UnrealObjectInfo.cs:line 93
   at ME3Explorer.Unreal.UnrealObjectInfo.GetEnumValues(MEGame game, String enumName, Boolean includeNone) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Unreal\ME3UnrealObjectInfo.cs:line 52
   at ME3Explorer.Unreal.EnumProperty..ctor(MemoryStream stream, IMEPackage pcc, NameReference enumType, Nullable`1 name) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Unreal\PropertyCollection.cs:line 910
   at ME3Explorer.Unreal.PropertyCollection.ReadProps(IMEPackage pcc, MemoryStream stream, String typeName) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Unreal\PropertyCollection.cs:line 158
   at ME3Explorer.Unreal.PropertyCollection.ReadArrayProperty(MemoryStream stream, IMEPackage pcc, String enclosingType, NameReference name, Boolean IsInImmutable) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Unreal\PropertyCollection.cs:line 458
   at ME3Explorer.Unreal.PropertyCollection.ReadProps(IMEPackage pcc, MemoryStream stream, String typeName) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Unreal\PropertyCollection.cs:line 172
   at ME3Explorer.Packages.ExportEntry.GetProperties() in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Packages\ExportEntry.cs:line 170
   at ME3Explorer.SequenceEditor.LoadObject(Int32 index) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Sequence Editor\SequenceEditor.cs:line 464
   at ME3Explorer.SequenceEditor.GenerateGraph() in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Sequence Editor\SequenceEditor.cs:line 431
   at ME3Explorer.SequenceEditor.LoadSequence(IExportEntry seqExport, Boolean fromFile) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Sequence Editor\SequenceEditor.cs:line 324
   at ME3Explorer.SequenceEditor.treeView1_AfterSelect(Object sender, TreeViewEventArgs e) in C:\Users\Mgamerz\Source\Repos\ME3Explorer\ME3Explorer\Sequence Editor\SequenceEditor.cs:line 301
   at System.Windows.Forms.TreeView.OnAfterSelect(TreeViewEventArgs e)
   at System.Windows.Forms.TreeView.TvnSelected(NMTREEVIEW* nmtv)
   at System.Windows.Forms.TreeView.WmNotify(Message& m)
   at System.Windows.Forms.TreeView.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```
Similar errors occur with other sequences.  The bug mentions "getEnumValues" one thing I noticed is that PMCheckConditional's & PMCheckState's & PMExecuteTransition's byteproperties "Conditional" "State" "Transition" "Region" "Plot" do not include "GlobalEnum" on their possible values even though that appears to be the default.

If you set all the above to allowed values and remove the BioSeqAct_ModifyPropertyPawn (which has lots of byteproperties too) from the sequence, it loads without issue

**Comment by Mgamerz** [comment]
Yes I am seeing this. I have seen this error before, and I know it is caught for ME3, in the code here it does not seem to be caught for ME1. I will look into why this enum is not being found.

**Comment by Mgamerz** [comment]
Please verify this is fixed in 3.0.9 so I can close this issue.

**Comment by Kinkojiro** [comment]
This is fixed in 3.0.9
